Rakefile

```diff
@@ -9,3 +10,36 @@ PuppetLint.configuration.send('disable_class_inherits_from_params_class')
 PuppetLint.configuration.send('disable_documentation')
 PuppetLint.configuration.send('disable_single_quote_string_with_variables')
 PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+default_acceptance_platform = 'windows2012r2-64'
+
+desc 'Generate Beaker Host config'
+rototiller_task :host_config, [:default_platform] do |t, args|
+  t.add_env({:name => 'PLATFORM', :message => 'PLATFORM Must be set. For example "windows2012r2-64"', :default => args[:default_platform]})
+  hosts_file = "spec/acceptance/nodesets/#{ENV['PLATFORM']}.yml"
+  t.add_command do |cmd|
+    cmd.name = 'bundle exec beaker-hostgenerator'
```

Comment:
Hooking directly into the API (https://github.com/puppetlabs/beaker-hostgenerator/blob/master/bin/beaker-hostgenerator#L8-L12, or even https://github.com/puppetlabs/beaker-hostgenerator/blob/a61487bf11b9df4b1706b9eabd3ea4801bac59f0/lib/beaker-hostgenerator/cli.rb#L161-L162) reduces the dependency on the environment (is everything running in bundler?) and would allow us to get rid of the shell hack in the next line.